src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -384,18 +383,18 @@
                             {
                                 if (@Model.DiffRevisionId == @revision.RevisionId)
                                 {
-                                    <option style="font-size: 0.8em;" value="@urlValue" selected data-content="@optionName"></option>
+                                        <option value="@urlValue" selected>@optionName</option>
                                 }
                                 else
                                 {
-                                    <option style="font-size: 0.8em;" value="@urlValue" data-content="@optionName"></option>
+                                        <option value="@urlValue">@optionName</option>

```

Comment:
Is the extra level of indention intentional?

Reply:
Nope, I have fixed it.